tensorbay/label/tests/test_label_polygon.py

```diff
@@ -109,3 +123,82 @@ def test_dumps(self, categories, attributes, subcatalog_polygon):
             del subcatalog_polygon["isTracking"]
 
         assert subcatalog.dumps() == subcatalog_polygon
+
+
+class TestLabeledMultiPolygon:
+    def test_init(self):
+        labeledmultipolygon = LabeledMultiPolygon(
+            [[(1.0, 2.0), (2.0, 3.0), (1.0, 3.0)], [(1.0, 4.0), (2.0, 3.0), (1.0, 8.0)]],
+            category="example",
+            attributes={"key": "value"},
+            instance="123",
+        )
+        assert labeledmultipolygon[0] == Polygon([(1.0, 2.0), (2.0, 3.0), (1.0, 3.0)])
+        assert labeledmultipolygon.category == "example"
+        assert labeledmultipolygon.attributes == {"key": "value"}
+        assert labeledmultipolygon.instance == "123"
+
+    def test_loads(self):
+
+        labeledmultipolygon = LabeledMultiPolygon.loads(_DATA_LABELEDMULTIPOLYGON)
+        assert labeledmultipolygon == LabeledMultiPolygon(
+            [[(1.0, 2.0), (2.0, 3.0), (1.0, 3.0)], [(1.0, 4.0), (2.0, 3.0), (1.0, 8.0)]],
+            category="example",
+            attributes={"key": "value"},
+            instance="123",
+        )
+        assert labeledmultipolygon.category == "example"
+        assert labeledmultipolygon.attributes == {"key": "value"}
+        assert labeledmultipolygon.instance == "123"
```

Comment:
Are these assertions still necessary once the `labeledmultipolygon` equals to the created `LabeledMultiPolygon` instance?

Reply:
Removed